CHANGES

```diff
@@ -70,6 +70,7 @@
     * Close Unix sockets if the connection attempt fails. This prevents `ResourceWarning`s. (#3314)
     * Close SSL sockets if the connection attempt fails, or if validations fail. (#3317)
     * Eliminate mutable default arguments in the `redis.commands.core.Script` class. (#3332)
+    * Allow never versions of PyJWT as dependency. (#3630)
```

Comment:
```suggestion
    * Allow newer versions of PyJWT as dependency. (#3630)
```

Reply:
Interesting typo. Telling a story of a long day
Thanks, fixed :)